packages/backend/src/invoice/service.test.ts

```diff
@@ -109,6 +134,60 @@ describe('Invoice Service', (): void => {
     })
   })
 
+  describe('deactivateNext', (): void => {
+    test('Does not deactivate a not-expired invoice', async (): Promise<void> => {
+      const invoiceId = (
+        await invoiceService.create({
+          paymentPointerId,
+          description: 'Test invoice',
+          expiresAt: new Date(Date.now() + 30_000)
+        })
+      ).id
+      await expect(invoiceService.deactivateNext()).resolves.toBeUndefined()
+      const invoice = await invoiceService.get(invoiceId)
+      if (!invoice) throw new Error('invoice was deleted')
+      expect(invoice.active).toBe(true)
+    })
+
+    test('Deactivates an expired invoice with received money', async (): Promise<void> => {
+      const invoice = await invoiceService.create({
+        paymentPointerId,
+        description: 'Test invoice',
+        expiresAt: new Date(Date.now() - 40_000)
+      })
+      const accountService = await deps.use('accountService')
+      const sourceAccount = await accountFactory.build({
+        balance: BigInt(10),
+        asset
+      })
+      const trxOrError = await accountService.transferFunds({
+        sourceAccount,
+        destinationAccount: invoice.account,
+        sourceAmount: BigInt(1),
+        timeout: BigInt(10e9) // 10 seconds
+      })
+      if (isAccountTransferError(trxOrError)) throw trxOrError
+      await expect(trxOrError.commit()).resolves.toBeUndefined()
+
+      await expect(invoiceService.deactivateNext()).resolves.toBe(invoice.id)
+      const invoiceAfter = await invoiceService.get(invoice.id)
+      if (!invoiceAfter) throw new Error('invoice was deleted')
+      expect(invoiceAfter.active).toBe(false)
+    })
+
+    test('Deletes an expired invoice with no money', async (): Promise<void> => {
+      const invoiceId = (
+        await invoiceService.create({
+          paymentPointerId,
+          description: 'Test invoice',
+          expiresAt: new Date(Date.now() - 40_000)
+        })
+      ).id
+      await expect(invoiceService.deactivateNext()).resolves.toBe(invoiceId)
+      expect(await invoiceService.get(invoiceId)).toBeUndefined()
```

Comment:
I tried this, and it looks like the invoice account isn't being deleted.
Adding `.onDelete('CASCADE')` [here](https://github.com/interledger/rafiki/blob/main/packages/backend/migrations/20211022210203_create_invoices_table.js#L5) didn't fix it for me.

```suggestion
      const { id: invoiceId, accountId } =
        await invoiceService.create({
          paymentPointerId,
          description: 'Test invoice',
          expiresAt: new Date(Date.now() - 40_000)
        })
      await expect(invoiceService.deactivateNext()).resolves.toBe(invoiceId)
      expect(await invoiceService.get(invoiceId)).toBeUndefined()
      const accountService = await deps.use('accountService')
      expect(await accountService.get(accountId)).toBeUndefined()
```